docs/command-reference/strings/bitop.md

```diff
@@ -10,36 +10,33 @@ import PageTitle from '@site/src/components/PageTitle';
 
 ## Introduction
 
-`BITOP` is a Redis/Dragonfly command used to perform bitwise operations between multiple keys and store the result in a destination key. It supports operations like AND, OR, XOR, and NOT. This command is crucial for handling binary data and performing efficient bulk bitwise operations.
+In Dragonfly, as well as in Redis and Valkey, the `BITOP` command is used to perform bitwise operations between multiple keys and store the result in a destination key.
+It supports operations like `AND`, `OR`, `XOR`, and `NOT`.
+This command is crucial for handling binary data and performing efficient bulk bitwise operations.
 
 ## Syntax
 
-```cli
+```shell
 BITOP operation destkey key [key ...]
 ```
 
 ## Parameter Explanations
 
-- **operation**: The bitwise operation to be performed. It can be `AND`, `OR`, `XOR`, or `NOT`.
-- **destkey**: The key where the result of the bitwise operation will be stored.
-- **key [key ...]**: One or more keys containing string values on which the bitwise operation will be applied.
+- `operation`: The bitwise operation to be performed. It can be `AND`, `OR`, `XOR`, or `NOT`.
```

Comment:
In `Parameter Explanations` sections, command/code related terms should be quoted instead of bold.